_specs/ecip-1068.md

```diff
@@ -0,0 +1,72 @@
+---
+lang: en
+ecip: 1068
+title: Account Versioning Extension for Contract Creation Transaction
+author: Wei Tang (@sorpaas)
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/169
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-06-24
```

Comment:
Since this is also being submitted as an ECIP with an Apache License, would it make sense to have created reflect the date of the submission of the ECIP itself?